src/CMakeLists.txt

```diff
@@ -678,9 +683,9 @@ add_executable(ceph-osd ${ceph_osd_srcs}
   $<TARGET_OBJECTS:common_util_obj>)
 add_dependencies(ceph-osd erasure_code_plugins)
 target_link_libraries(ceph-osd osd os global ${BLKID_LIBRARIES})
-if(WITH_FUSE)
+if(HAVE_LIBFUSE)
```

Comment:
`HAVE_LIBFUSE` is for the subvar in `acconfig.h`. we'd better use `WITH_*` for checking the options.

and also, i don't think the purpose of this PR is to consolidate these two variants. also you are not touching this piece of code. so could you please leave it as it is?